changelog.txt

```diff
@@ -1,5 +1,18 @@
 == Changelog ==
 
+= 0.5.0 =
+* Fix redirect CPT to be private
+* Add filters to modify the contents of the HTTP Status Code drop down enhancement good first issue
```

Comment:
Not really sure what "good first issue" has to do here.

Reply:
copy paste error - those are the github labels, I forgot to remove them